scout/server/blueprints/dashboard/templates/dashboard/dashboard_general.html

```diff
@@ -1,100 +1,62 @@
+        <!--Search institute select-->
+        <div class="col-2 ml-3">
+          {{ dashboard_form.search_institute(class="form-control") }}
+        </div>
 
-{% macro dashboard_search_form(pane) %}
-<br>
-<form id='form' class="form-horizontal" method='POST' action="{{ url_for('dashboard.index') }}" accept-charset="utf-8">
-    <input type="hidden" value={{pane}} name="pane_id" id="pane_id">
-    <div class="row text-center">
-      <div class="col-md-4 col-sm-4">
-        {% if pane in (1,2) %}
-          <div class="input-group">
-            <span class="input-group-addon">
-              <span class="glyphicon glyphicon-search"></span>
-            </span>
-            <input type="search" class="form-control" value="{{ query if query }}" name="query" id="query" placeholder="Enter query to filter cases summarized below" onchange="update_select({% if current_user.is_admin %}true{% else %}false{% endif %})"></input>
+        {% if panel != "3" %}
+          <!--Search type select field-->
+          <div class="col-3 ml-3">
+            {{ dashboard_form.search_type(class="form-control", id="search_type") }}
+          </div>
+          <!--Search term textfield-->
+          <div class="col-4 ml-3">
+            {{ dashboard_form.search_term(class="form-control", placeholder="Search term", id="search_term") }}
           </div>
         {% endif %}
-      </div>
-      <div class="col-md-4 col-sm-2">
-      <select name="institute" class="form-control" id="institute" onchange="this.form.submit()">
-        {% for inst in institutes %}
-          {% if inst.display_name == 'All institutes' and query and not current_user.is_admin %}
-            <option value="{{ inst._id }}" disabled>{{ inst.display_name }}</option>
-          {% else %}
-            <option value="{{ inst._id }}" {{ "selected" if inst._id == choice }} >{{ inst.display_name }}</option>
-          {% endif %}
-        {% endfor %}
-      </select>
-      </div>
-      <div class="col-md-2 col-sm-2">
-        <button type="submit" class="form-control">Search</input>
-      </div>
-    </div>
-    {% if pane in (1,2) %}
-    <br>
-    <div class="bottom-align-text">
-      Query examples:
-      {% for elem in ["case:18201", "exact_pheno:HP:0001166", "synopsis:epilepsy", "panel:NMD", "pheno_group:HP:0001250", "status:active", "cohort:pedhep", "similar_case:1820", "similar_pheno:HP:0001166,HP:0001250", "pinned:POT1", "causative:POT1", "user:Kent"]%}
-        <span class="badge badge-info">{{elem}}</span>
-      {% endfor %}
-    </div>
-    {% endif %}
-  </form>
-  <br>
+
+        <!--Search term textfield-->
+        <div class="col-1 ml-3">
+          {{ dashboard_form.search(class="form-control") }}
+        </div>
+
+      </div> <!--end of class="form-row align-items-center"-->
+    </form>
+  </div>
 {% endmacro %}
 
 {% macro general_stats_panels() %}
-<div class="mt-3 text-center">
+<div class="mt-3 text-center" id=1>
```

Comment:
Would it be possible to have the ids more verbose? Tracking 1,2,3 through the code.. 😬

Reply:
I had them like this because I loop from 1 to 3 in javascript in the function that hides/shows the divs, but of course it's clearer using more descriptive names, I'll fix it!